tests/glimmer/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -872,6 +872,16 @@ printWidth: 80
     ac
   {{/if}}
 {{/if}}
+
+{{#if a}}
+  a
+{{else}}
+  {{#if c}}
+    c
+  {{/if}}
+  e
+{{/if}}
+
```

Comment:
Please add tests where in `if`/`else` more then one character, like:
```
{{#if a}}
  a
  b
  c
{{else}}
  {{#if c}}
    a
    b
    c
  {{/if}}
  a
  b
  c
{{/if}}
```

Reply:
This is done 👍